docs/using_eessi/eessi_container.md

```diff
@@ -0,0 +1,165 @@
+# Using the EESSI container
+
+This page provides all information and guides you through several examples
+illustrating to access the EESSI software stack via a small container provided by
+EESSI. The container only contains a minimal number tools such as the CernVM-FS
+client and some configuration data.
+
+### Prerequisites
+
+- Singularity 3.7.x or newer _or_ Apptainer x.y.z or newer (any version with
+  support for `--fusemount` should be sufficient)
+- `git` available on your system
+
+### Preparation
+
+Clone the [`EESSI/software-layer`](https://github.com/EESSI/software-layer.git)
+repository to a directory `eessi_container` by running the command
+```bash
+git clone https://github.com/EESSI/software-layer.git eessi_container
+```
+Change directory
+```bash
+cd eessi_container
+```
```

Comment:
I wouldn't rename `software-layer`, only creates confusion. Also, make it a single box to copy-paste easily?

``````suggestion
Clone the [`EESSI/software-layer`](https://github.com/EESSI/software-layer.git)
repository, and change into the resulting `software-layer` directory:
```bash
git clone https://github.com/EESSI/software-layer.git
cd software-layer
```
``````

Reply:
Fixed by https://github.com/EESSI/docs/pull/100/commits/0eae16fe58096e7b995d7e7d2ddc3d9208fc2001